src/nvim/api/ui_events.in.h

```diff
@@ -89,7 +89,7 @@ void grid_scroll(Integer grid, Integer top, Integer bot, Integer left, Integer r
                  Integer cols)
   FUNC_API_SINCE(5) FUNC_API_REMOTE_IMPL FUNC_API_COMPOSITOR_IMPL;
 void grid_destroy(Integer grid)
-  FUNC_API_SINCE(6) FUNC_API_REMOTE_ONLY;
+  FUNC_API_SINCE(6) FUNC_API_MULTIGRID_IMPL;
```

Comment:
what is the intended meaning of this new name?

Reply:
It's for events only sent to multigrid UIs. The naming can probably be changed, I adapted it from ´FUNC_API_REMOTE_IMPL´ and `FUNC_API_COMPOSITOR_IMPL`

Reply:
> events only sent to multigrid UIs

Is that actually important? E.g. in this case, for a non-multigrid UI, it either won't ever see `grid_destroy` or if it does, what harm would it cause?

If this is only to save RPC traffic, let's drop it. 

We plan to remove `FUNC_API_REMOTE_ONLY`, for example: https://github.com/neovim/neovim/issues/28437
Avoiding special distinction is helpful (similar to the purpose of this PR).

Reply:
I guess it doesn't cause any harm. but it could be confusing to receive a multigrid event that doesn't necessarily contain the correct information, especially if the rest of the of the PR isn't merged, so that the correct information to include in the events isn't even calculated.

The events themselves are not that big, so the bandwidth probably doesn't matter too much.

If we want to prevent the events, like we currently do for most multigrid events, then this enables mixing multigrid and non-multigrid, and still have the events only delivered to the UIs that support it. It also removes the need to check if a multigrid ui is connected or not, all over the code case.

Reply:
> It also removes the need to check if a multigrid ui is connected or not, all over the code

Oh, that sounds valuable :)

Reply:
> It also removes the need to check if a multigrid ui is connected or not, all over the code

Oh, that sounds valuable :)